CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# CODEOWNERS
+
+* @opentdf/nifi @opentdf/architecture
```

Comment:
looks like we don't have a nifi team

Reply:
Going to follow up on this with a PR if necessary.  I thought our terraform teams had this defined appropriately.